tests/integration/test_integration.py

```diff
@@ -745,21 +746,29 @@ def test_downgrade(encoding, monkeypatch):
     log.error.assert_not_called()
 
 
-@allencodings
-def test_span_tags(encoding, monkeypatch):
-    monkeypatch.setenv("DD_TRACE_API_VERSION", encoding)
+def test_span_tags(run_python_code_in_subprocess):
```

Comment:
you could keep this with `@allencodings`, and then just use it to set the `DD_TRACE_API_VERSION` env var for the subprocess call.